app/utils/defaultSettings.js

```diff
@@ -25,6 +25,7 @@ module.exports = {
   posLatitude: 0.0,
   posLongitude: 0.0,
   useMonochromeTrayIcon: true,
+  silentNotifications: true,
```

Comment:
by default should be false, so we do not change the existing behavior

Reply:
Fixed!